config/locales/en.yml

```diff
@@ -1056,7 +1056,7 @@ en:
       money_will_be_transferred: "The money will be transferred to %{recipient_name} when a) you have marked the request completed or b) %{automatic_confirmation_days} days have passed since you paid."
       listing_per_unit_title: "%{title}, per %{unit_type}"
       stripe:
-        you_have_made_new_payment: "You have paid <b>%{payment_sum}</b> for <b>%{listing_title}</b> to %{service_name}. The money will be released to %{recipient_full_name} once you mark the order as completed. Here is a receipt of the payment."
+        you_have_made_new_payment: "You have paid <b>%{payment_sum}</b> for <b>%{listing_title}</b>. The money is being held by %{service_name} and will be released to %{recipient_full_name} once you mark the order as completed. Here is a receipt of the payment."
```

Comment:
The "The money is being held by %{service_name}" part is not fully correct as it's actually held by Stripe, but not really the marketplace or marketplace admins. But I guess that's not super important, I don't have any better wording in mind.